lib/skate/detours/detours.ex

```diff
@@ -197,20 +198,67 @@ defmodule Skate.Detours.Detours do
   def update_or_create_detour_for_user(user_id, uuid, attrs \\ %{}) do
     user = User.get_by_id!(user_id)
 
-    case uuid do
-      nil ->
-        create_detour_for_user(user_id, attrs)
-
-      _ ->
-        Repo.insert(
-          Detour.changeset(%Detour{author: user, id: uuid}, attrs),
-          returning: true,
-          conflict_target: [:id],
-          on_conflict: {:replace, [:state, :updated_at]}
-        )
+    previous_state = categorize_detour_by_id(uuid, user_id)
```

Comment:
non-blocking suggestion. Could this be renamed, since state is already a different "thing" in detour-land?
```suggestion
    previous_status = categorize_detour_by_id(uuid, user_id)
```

Reply:
https://github.com/mbta/skate/compare/9dd70f4cdb65c0fcbcdf5091ac55ac4199d57cc5..9eeaca182018d091dbc17c5afb4712215a4d96d2